feature/community/src/main/java/com/iteneum/community/domain/CommunityCardDataModel.kt

```diff
@@ -0,0 +1,13 @@
+package com.iteneum.community.domain
+
+import androidx.compose.ui.graphics.vector.ImageVector
+
+/**
+ * this Data class represents the model for each button inside UI view Community
+ * where every instance will turn into a clickable button
+ * @param icon needs an Image vector to set the icon of the card Button
+ * @param cardName  uses the Enum list to set the propper name for the Card Button
+ *
+ * @author Andrés Ivan Medina Herrera
+ */
+data class CommunityCardDataModel(val icon: ImageVector, val cardName: CommunitySectionList)
```

Comment:
If you use a sealed class instead an Enum, you don't need to implement all this stuff and your code would be readable and easier to understand also you'll be able to avoid boilerplate and it would look more elegant.

Reply:
@irv205  will it be or will not be implemented  get cleared to proceed on the fix please

Reply:
I still think you don't need acceptance criteria to create a sealed class, I mean, did you needed acceptance criteria to create a enum class? I just wanted to implement the sealed class because is one of the features of kotlin and would be easier and less rewrite when we'll decide to get the data from the server. Also because is important to you learn it and talk in the interviews with more confidence.